src/extension.ts

```diff
@@ -54,7 +54,11 @@ export async function activate(context: vscode.ExtensionContext) {
       octokit = await credentials.getOctokit();
 
       getPRsPerSHAs();
-      searchType({ searchType: "watermelon.start", owner, repo, localUser });
+
+      // TODO: Handle case where user is not authenticated
```

Comment:
That does not exist.
Everytime the extension is activated it creates the auth object.
It simply cannot be used without auth.